doc/build-instructions/cmake.md

```diff
@@ -40,6 +40,8 @@ can be followed with the following modifications:
 :pencil: For more CMake info see [OMR CMake documentation](https://github.com/eclipse/omr/blob/master/doc/BuildingWithCMake.md)
 
 ## FAQ
+[Why are only some targets installed via `install()`?](#why-are-only-some-targets-installed-via-install)
+[How do I add build flags?](#how-do-i-add-build-flags)
```

Comment:
When I view this file, this link comes out on the same line as the previous link.